pkg/runtime/runtime.go

```diff
@@ -68,6 +69,30 @@ func (r *Runtime) formatUiOptions() (*string, error) {
 		modules[i] = b.ValueOf()
 	}
 
+	appBase := map[string]interface{}{}
+	appBaseBuf, err := os.ReadFile(fmt.Sprintf("%v/app.base.json", r.patchDir))
```

Comment:
how about saving the appBaseHash in the patch file inline?

Reply:
Done.